tests/hook/call.c

```diff
@@ -0,0 +1,71 @@
+#include "clar_libgit2.h"
+
+static git_repository *repo = NULL;
```

Comment:
No need for this static initialization

Reply:
That's the "standard" test case static repo (albeit not named g_repo) 😕. I'll just rename it then.